### ISSUE TITLE
Error when using vakt.rules.string.RegexMatchRule

### ISSUE THREAD

**Issue by skotti** [kolotaev/vakt#7]
Hello!
I want to write this kind of policy:
```
Policy(
        uid = str(8),
        effect = DENY_ACCESS,
        subjects = ['<.*>'],
        resources = ['vms'],
        actions = ['pause'],
        rules = {
            'vm': vakt.rules.string.RegexMatchRule('m.*'),
        }
    )
```

And I get this kind of error:
```

Traceback (most recent call last):
  File "creator.py", line 54, in <module>
    st.add(p)
  File "/home/skotti/attribute_conf/vakt/vakt/storage/mongo.py", line 36, in add
    self.collection.insert_one(self.__prepare_doc(policy))
  File "/home/skotti/attribute_conf/vakt/vakt/storage/mongo.py", line 110, in __prepare_doc
    doc = b_json.loads(policy.to_json())
  File "/home/skotti/attribute_conf/vakt/vakt/util.py", line 33, in to_json
    default=lambda o: o.to_json() if isinstance(o, JsonSerializer) else vars(o))
  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)
  File "/usr/lib/python3.5/json/encoder.py", line 198, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.5/json/encoder.py", line 256, in iterencode
    return _iterencode(o, 0)
  File "/home/skotti/attribute_conf/vakt/vakt/util.py", line 33, in <lambda>
    default=lambda o: o.to_json() if isinstance(o, JsonSerializer) else vars(o))
  File "/home/skotti/attribute_conf/vakt/vakt/util.py", line 33, in to_json
    default=lambda o: o.to_json() if isinstance(o, JsonSerializer) else vars(o))
  File "/usr/lib/python3.5/json/__init__.py", line 237, in dumps
    **kw).encode(obj)
  File "/usr/lib/python3.5/json/encoder.py", line 198, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.5/json/encoder.py", line 256, in iterencode
    return _iterencode(o, 0)
  File "/home/skotti/attribute_conf/vakt/vakt/util.py", line 33, in <lambda>
    default=lambda o: o.to_json() if isinstance(o, JsonSerializer) else vars(o))
```
This happens only with this regular expressions, for example with StringEqualRule everything is fine.

**Comment by kolotaev** [comment]
@skotti, thanks for the issue.
We'll look ASAP. Fix will be released in 1.1.1 as soon as it's merged.

**Comment by kolotaev** [comment]
Ah, I see, this is a known issue and is already fixed in [attribute-checker](https://github.com/kolotaev/vakt/tree/attribute-checker) branch which is going to become v1.2.0.
I'll back-port it to 1.1.1.
